vue-frontend/src/components/home-new/LandingSection.vue

```diff
@@ -15,13 +15,19 @@
 
     <div class="container">
       <div class="flex-div">
-        <div class="image">
+        <picture class="image">
+          <source
+            srcset="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs="
```

Comment:
How we have got `base64` data?

Reply:
It's basically the smallest empty image.

Reply:
Got from here https://swimburger.net/blog/web/web-performance-prevent-wasteful-hidden-image-requests